resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java

```diff
@@ -111,6 +114,32 @@ public Map<String, String> getCommonTemplateVariables(String eventId) {
         return templateVariables;
     }
 
+    private String truncateMilliSecondInTimeStampString(LocalDateTime localDateTime) {
```

Comment:
For the truncateMilliSecondInTimeStampString please use date-time formatting instead of string parsing..

Reply:
done.